extension/js/common/helpers.ts

```diff
@@ -164,3 +164,11 @@ export const processAndStoreKeysFromEkmLocally = async (
   }
   return { updateCount: encryptedKeys?.keys.length ?? 0 + (existingKeys.length - keysToRetain.length), noKeysSetup: !(encryptedKeys?.keys.length || keysToRetain.length) };
 };
+
+export const getLocalKeyExpiration = async ({ acctEmail }: Bm.GetLocalKeyExpiration): Promise<Bm.Res.GetLocalKeyExpiration> => {
+  const keys = await KeyStore.get(acctEmail);
+  const keyInfos = await Promise.all(keys.map(async (key) => await KeyUtil.parse(key.public)));
+  const keyExpiration = keyInfos
+    .sort((key1, key2) => (key1.expiration ?? 0) < (key2.expiration ?? 0) ? 1 : -1)[0]?.expiration;
```

Comment:
Shouldn't `undefined` expiration be treated like `Infinity`?

Reply:
Nice catch. Yeah, it should be treated as Infinity.